City-of-Helsinki/drupal-infofinland
# UHF-9631

Remove previously disabled modules

## What was done
- Added temporary select2_icon module to allow removing the hdbt_admin-theme
- Removed hdbt_admin -theme
- Removed custom ptv-integration module.
- Added D10 to custom module's core version requirement

## How to install

* Make sure your instance is up and running on correct branch.
  * `git checkout UHF-9631`
  * `make fresh`
* Run `make drush-cr`

## How to test
The pr mainly affects admin-side. Go and edit nodes and save nodes. Everything should work as before.
